src/components/Modals/Onboarding/OnboardingTransfer.vue

```diff
@@ -225,7 +162,7 @@ export const getExamCredit = (exam: FirestoreAPIBExam): number => {
 
 export default Vue.extend({
   components: {
-    CourseSelector,
+    // CourseSelector,
```

Comment:
If we don't use CourseSelector then I would just remove this instead of commenting it out.

Reply:
missed this when removing comments

Reply:
<img width="636" alt="Screen Shot 2021-03-26 at 11 14 35 AM" src="https://user-images.githubusercontent.com/14082828/112653071-75211000-8e24-11eb-803b-542da12a2a09.png">